mypy/build.py

```diff
@@ -396,7 +396,9 @@ def default_lib_path(data_dir: str,
                         ('hash', str),
                         ('dependencies', List[str]),  # names of imported modules
                         ('data_mtime', int),  # mtime of data_json
+                        ('deps_mtime', Optional[int]),  # mtime of deps_json
                         ('data_json', str),  # path of <id>.data.json
+                        ('deps_json', Optional[str]),  # path of <id>.deps.json
```

Comment:
Also add comment with hint about what the `...deps.json` files is used for (fine-grained deps?).